editoast/src/views/timetable/stdcm.rs

```diff
@@ -283,6 +324,13 @@ async fn stdcm(
     }
 }
 
+async fn log_stdcm(conn: &mut DbConnection, stdcm_log_changeset: Changeset<StdcmLog>) {
```

Comment:
Not sure that I understand what I'm expected for. If the idea is to put more in the function, less in the closure, yes, it does make sense.

Reply:
It's more about using the new db connection vs opening a new one.

* If we open a new connection, we need the function to simplify the closure as there's two sources of error (`get` and `create`). But we may end up with the same deadlocking issues we encountered a while ago.
* If we clone the existing connection, we don't need the function since there's only one source of error anymore: `create`, so we can leave it in the closure.

Reply:
As discussed, I deleted the function and fixed the borrowing issue with the connection.